.github/workflows/check.yml

```diff
@@ -123,6 +123,6 @@ jobs:
     - uses: actions/setup-python@v4
```

Comment:
Can't add a suggestion so far up, but this job is still named `bindcheck`. Maybe it should be renamed to `flake8-tkinter` or `flake8`.